cmd/submit_test.go

```diff
@@ -534,6 +537,46 @@ func TestSubmitRelativePath(t *testing.T) {
 	assert.Equal(t, "This is a file.", submittedFiles["file.txt"])
 }
 
+func TestSubmitServerErr(t *testing.T) {
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprintf(w, `{"error": {"type": "error", "message": "test error"}}`)
+	})
+
+	errts := httptest.NewServer(handler)
```

Comment:
We only have one test server in this test, right? If so, can we name the variable `ts` for consistency with the other tests? By giving it a different name I find myself reading everything more carefully to see in what way it is special or what the other servers in the test are called to require it to be special.